Sources/TestingInternals/include/Discovery.h

```diff
@@ -53,7 +53,7 @@ SWT_EXTERN void swt_enumerateTypes(
   void *_Null_unspecified context,
   SWTTypeEnumerator body,
   SWTTypeNameFilter _Nullable nameFilter
-) SWT_SWIFT_NAME(swt_enumerateTypes(_:_:withNamesMatching:));
+) SWT_SWIFT_NAME(enumerateTypes(_:_:withNamesMatching:));
```

Comment:
Sort of unrelated, but now this is making me wonder why we don't give the first parameter a label like `context:`, to help explain its purpose?